stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/composer/MessageInputState.kt

```diff
@@ -9,9 +9,11 @@ import io.getstream.chat.android.common.state.MessageAction
  * @param inputValue The current text value that's within the input.
  * @param attachments The currently selected attachments.
  * @param action The currently active [MessageAction].
+ * @param maxMessageLength The maximum allowed message length.
  */
 public data class MessageInputState(
     val inputValue: String = "",
     val attachments: List<Attachment> = emptyList(),
     val action: MessageAction? = null,
+    val maxMessageLength: Int = Integer.MAX_VALUE,
```

Comment:
Does our BE support this kind of a limit? Maybe it's better to set it to a default (5000) just to make sure the message API will work when the users try to send a message.